.github/workflows/integration.yml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 jobs:
     integration:
         name: Integration tests
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-latest
```

Comment:
You have to download mysqldump tool when running on latest Ubuntu release ;)

Reply:
Oh... Thanks ;)

Reply:
That doesn't seem the only need

Reply:
Maybe some dependencies are missing, it's time to debug it :sweat_smile: